test/Base_Tests/src/Data/Numbers_Spec.enso

```diff
@@ -565,6 +571,16 @@ add_specs suite_builder =
             3.truncate . should_equal 3
             -3.truncate . should_equal -3
 
+    suite_builder.group "Hashing" group_builder->
+        # TODO: Pending https://github.com/enso-org/enso/issues/9296.
```

Comment:
```suggestion
        # TODO: Pending https://github.com/enso-org/enso/issues/9296
```

The dot at the end makes the link unclickable, at least from GH

Reply:
Done